src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1452,6 +1452,11 @@ To create/register a plugin, you have to :
     #define USES_C013   // ESPEasy P2P network
 #endif
 
+#ifdef FEATURE_CAN
+    #define USES_C020 //CAN
+    #define USES_C021
+    #define USES_P155
+#endif
```

Comment:
This should be like:
```
#ifndef FEATURE_CAN
  #define FEATURE_CAN 1
#endif
#if FEATURE_CAN
  #ifndef USES_C022 // CAN Controller
    #define USE_C022
  #endif
  #ifndef USES_P174
    #define USES_P174 // CAN Import plugin
  #endif
#endif
```
(For Plugin, Controller and Notifiers we still use `#ifdef USES_...` checks...)
Not sure why you also try to enable Controller C021? If that's one you also created yourself, then it should probably get ID C023, as C021 is reserved for a (planned) LoRa controller 🤔